fasta/pFasta/tools_test.go

```diff
@@ -0,0 +1,314 @@
+package pFasta
+
+import (
+	"fmt"
+	"testing"
+
+	"golang.org/x/exp/rand"
+
+	"github.com/vertgenlab/gonomics/bed"
+	"github.com/vertgenlab/gonomics/dna"
+	"github.com/vertgenlab/gonomics/dna/pDna"
+	"github.com/vertgenlab/gonomics/fasta"
+)
+
+// ExtractTestsOne tests a valid input to Extract
+var ExtractTestsSuccess = []struct {
```

Comment:
Comment name for this var disagrees with name here. I would go with ExtractTests if possible

Reply:
Sounds good! Changed for ExtractTests and ExtractBedTests